framework/src/bundle/BundleRegistry.cpp

```diff
@@ -52,9 +52,7 @@ class InitialBundleMapCleanup
   InitialBundleMapCleanup(std::function<void()> cleanupFcn)
```

Comment:
Just checking to make sure you meant to include this file with your submission. Looks like it's part of your other change?